test/unit/test-main.js

```diff
@@ -31,12 +31,15 @@ requirejs.config({
         bootstrap: 'ext_components/bootstrap/dist/js/bootstrap.min',
         testUtil: '../../test/unit/testUtil',
         narrativeMocks: '../../test/unit/mocks',
+        test: '../../test',
         bluebird: 'ext_components/bluebird/js/browser/bluebird.min',
         jed: 'components/jed/jed',
         custom: 'kbase/custom',
         // Note that this is only provided in the testing amd module configuration;
         // Mock Service Worker (MSW) is not active in the actual Narrative web app.
         msw: 'ext_modules/msw/index',
+        // React: 'ext_modules/react/react.development',
+        // ReactDOM: 'ext_modules/react/react-dom.development',
```

Comment:
Still iterating on this, I haven't asked for a review yet, but thanks!
I need the test and build to work so I can push an image up, so the requestor can look at how it works and we can iterate on Changes.

Reply:
Yeah, I'll do that.
I hadn't because Bill and I have been discussing this PR, and he was awaiting my go ahead after recasting it from a fork PR to a branch PR. I didn't expect you to jump on it :)
It was just revived from a 2019 PR he had asked me to hold off on until preact had been replaced by React in an upcoming Jupyter notebook update.